src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/1.sql

```diff
@@ -1278,22 +1278,7 @@ CREATE UNIQUE NONCLUSTERED INDEX IX_ExportJob_Status_HeartbeatDateTime_QueuedDat
 GO
 
 /*************************************************************
-    Sequence for generating unique 12.5ns "tick" components that are added
-    to a base ID based on the timestamp to form a unique resource surrogate ID
-**************************************************************/
-
-CREATE SEQUENCE dbo.ResourceSurrogateIdUniquifierSequence
-        AS int
-        START WITH 0
-        INCREMENT BY 1
-        MINVALUE 0
-        MAXVALUE 79999
-        CYCLE
-        CACHE 1000000
-GO
-
-/*************************************************************
-    Stored procedure for exporting
+    Stored procedures for exporting
```

Comment:
Because this is all in one big file, I thing it would be good to put the definitions related to export after those related to CRUD, so I suggest moving these to the bottom of the file.

Reply:
Right thanks, makes sense. Moved in ff5f3e1.